test/unit-tests/TestWithSplunk.cs

```diff
@@ -355,9 +355,20 @@ public static void StallWrite()
                 request.Headers.Add("Authorization", "Splunk " + token);
                 request.ContentType = "application/x-www-form-urlencoded";
                 request.ContentLength = byteArray.Length * 2;
-                Stream dataStream = request.GetRequestStream();
-                streams.Add(dataStream);
-                dataStream.Write(byteArray, 0, byteArray.Length);
+                try
```

Comment:
Reduced the iteration count and added a restart splunk when a Timeout is encountered.